__test__/supertest.test.js

```diff
@@ -1,34 +1,50 @@
 const request = require('supertest');
 const app = require('../app');
-
+// importent!!!!!!
 // befor run the test go to url-shortener/api/shorturl and change in the task.js the line 13 to 'const jsonFileName = "index2"' 
 
 
 describe("get method",()=>{
```

Comment:
Very nice, missing `400` test case here for `redirect`, and `400` and `404` for `statistic`